### DISCUSSION TITLE
[Feature request] sequelize 启动时, 链接失败不要导致全局退出

### DISCUSSION THREAD

**Discussion by geekberryonekey** [midwayjs/midway#3992] [Ideas]
### The feature, motivation（功能、动机）

在多数据源的情况下, 有的项目在部分数据源连接成功时仍然可以工作.

但是目前的链接会强制检查左右链接是否成功, 导致无法实现上述逻辑

### Alternatives（替代方案）

https://github.com/midwayjs/midway/blob/eed30f2d06fd957a80e1c24402caf11008ae05ed/packages/sequelize/src/dataSourceManager.ts#L50

将这个位置的强制校验换成日志输出, 如:
```ts
// await client.authenticate();
await this.checkConnected(client);
```

### Additional context（其他上下文）

@midwayjs/sequelize 3.15.2

**Comment by czy88840616** [comment]
一般启动的时候不成功就不会让启了吧。。

**Reply by geekberryonekey** [reply]
现在的业务需求是多个数据库源, 希望部分链接失败不会影响其他数据源的服务, 失败的可以慢慢修复

**Reply by czy88840616** [reply]
我想想，应该可以加个开关

**Reply by czy88840616** [reply]
https://github.com/midwayjs/midway/pull/3993 这个 pr 是处理连接验证的部分，同时和原有的 validateConnection 配置一起使用就可以满足。

**Reply by geekberryonekey** [reply]
请问 v4 中将 validateConnection 废弃, 部分连接成功时是否可以启动?

**Reply by czy88840616** [reply]
现在默认不会校验链接，所以可以启动。如有强校验的需求，可以显式声明 validateConnection 配置。
